search/build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
+
+dependencies {
+
+    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
+    implementation 'com.android.support:support-v4:27.1.1'
```

Comment:
This should use the same `libs` constant defined in the root directory's `build.gradle` file.

Reply:
Now uses the `constraint_layout` defines in the root `build.gradle`.